src/libs/navigation/Link.tsx

```diff
@@ -19,7 +19,7 @@ export const Link = function Link({ to, action, target, onPress, children, param
 
   return (
     <DefaultLink
-      to={`${to}${params ? `?${searchParams.toString()}` : ''}`}
+      to={`${to && to[0] !== '/' ? `/${to}` : to}${params ? `?${searchParams.toString()}` : ''}`}
```

Comment:
`to` est `required` d'après le typage + changer la condition en positif : 
```suggestion
      to={`${to[0] === '/' ? to : `/${to}`}${params ? `?${searchParams.toString()}` : ''}`}
```

Reply:
ok